attributes/default.rb

```diff
@@ -80,6 +80,9 @@
 # Enable the Chef handler to report to datadog
 default['datadog']['chef_handler_enable'] = true
 
+# List to notify in case of chef failures
+default['datadog']['handler']['notify_on_failure'] = []
```

Comment:
Should the default be an array? I think it should be `nil` and activated with `true`